caffe2/sgd/adadelta_op.h

```diff
@@ -0,0 +1,179 @@
+#pragma once
+
+#include "caffe2/core/operator.h"
+
+namespace caffe2 {
+
+template <typename Context>
```

Comment:
Make such internal function in an unnamed namespace. And also please make the function names follow https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions

Reply:
Sorry i didn't get this. I think I followed the template as in other ops (like adagrad).